<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BuffWgt</class>
 <widget class="QWidget" name="BuffWgt">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>876</width>
    <height>472</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="2" column="0" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>30</height>
        </size>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>448</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="m_pBtnSeid">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>30</height>
        </size>
       </property>
       <property name="text">
        <string>Seid管理</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="m_pBtnOK">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>30</height>
        </size>
       </property>
       <property name="text">
        <string>确定</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="m_pBtnCancel">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>30</height>
        </size>
       </property>
       <property name="text">
        <string>取消</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="3" column="0">
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="0">
    <widget class="QWidget" name="widget" native="true">
     <property name="maximumSize">
      <size>
       <width>150</width>
       <height>16777215</height>
      </size>
     </property>
     <layout class="QGridLayout" name="gridLayout_4">
      <item row="2" column="0" colspan="3">
       <widget class="QPushButton" name="m_pBtnCustomIcon">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>30</height>
         </size>
        </property>
        <property name="text">
         <string>自定义图标</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <spacer name="horizontalSpacer_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>13</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="0" column="2">
       <spacer name="horizontalSpacer_4">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>18</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="1" column="0" colspan="3">
       <widget class="Tool_NumberEdit" name="BuffIcon">
        <property name="minimumSize">
         <size>
          <width>128</width>
          <height>30</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>128</width>
          <height>16777215</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="m_pLabIcon">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>64</width>
          <height>64</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>64</width>
          <height>64</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(255, 255, 255);</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="3" column="0" colspan="3">
       <widget class="QPushButton" name="m_pBtnDelCustomICon">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>30</height>
         </size>
        </property>
        <property name="text">
         <string>清除自定义图标</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="1" rowspan="2">
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="minimumSize">
        <size>
         <width>120</width>
         <height>30</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>120</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>ID</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="Tool_NumberEdit" name="buffid">
       <property name="minimumSize">
        <size>
         <width>200</width>
         <height>30</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="0" column="2" colspan="2">
      <widget class="QLabel" name="label_3">
       <property name="minimumSize">
        <size>
         <width>120</width>
         <height>30</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>120</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>名称</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="0" column="4">
      <widget class="QLineEdit" name="name">
       <property name="minimumSize">
        <size>
         <width>200</width>
         <height>30</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_5">
       <property name="minimumSize">
        <size>
         <width>120</width>
         <height>30</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>120</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>类型</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QComboBox" name="bufftype">
       <property name="minimumSize">
        <size>
         <width>200</width>
         <height>30</height>
        </size>
       </property>
       <item>
        <property name="text">
         <string>负面Buff</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>正面Buff</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>指示物</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>阵法</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>形态</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>功法被动</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>装备被动</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>丹药</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>结婴</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>结丹</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>暗劲</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>悟道</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>元婴</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>筑基</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>化神</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="1" column="2" colspan="2">
      <widget class="QLabel" name="label_2">
       <property name="minimumSize">
        <size>
         <width>120</width>
         <height>30</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>120</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>buff特效</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="1" column="4">
      <widget class="QLineEdit" name="skillEffect">
       <property name="minimumSize">
        <size>
         <width>200</width>
         <height>30</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_4">
       <property name="minimumSize">
        <size>
         <width>120</width>
         <height>30</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>120</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>词缀[,]</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="Tool_ListEdit" name="Affix">
       <property name="minimumSize">
        <size>
         <width>200</width>
         <height>30</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_6">
       <property name="minimumSize">
        <size>
         <width>120</width>
         <height>30</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>120</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>buff效果（Seid）[,]</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_7">
       <property name="minimumSize">
        <size>
         <width>120</width>
         <height>30</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>120</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>描述</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="4" column="1" colspan="4">
      <widget class="QLineEdit" name="descr">
       <property name="minimumSize">
        <size>
         <width>150</width>
         <height>30</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="label_8">
       <property name="minimumSize">
        <size>
         <width>120</width>
         <height>30</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>120</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>触发时机</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QComboBox" name="trigger">
       <property name="minimumSize">
        <size>
         <width>200</width>
         <height>30</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="5" column="2" colspan="2">
      <widget class="QLabel" name="label_9">
       <property name="minimumSize">
        <size>
         <width>120</width>
         <height>30</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>120</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>移除时机</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="5" column="4">
      <widget class="QComboBox" name="removeTrigger">
       <property name="minimumSize">
        <size>
         <width>200</width>
         <height>30</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="label_13">
       <property name="minimumSize">
        <size>
         <width>120</width>
         <height>30</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>120</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>叠加类型</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="QComboBox" name="BuffType">
       <property name="minimumSize">
        <size>
         <width>200</width>
         <height>30</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="6" column="2" colspan="2">
      <widget class="QLabel" name="label_15">
       <property name="minimumSize">
        <size>
         <width>120</width>
         <height>30</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>120</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>显示层数</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="6" column="4">
      <widget class="QComboBox" name="ShowOnlyOne">
       <property name="minimumSize">
        <size>
         <width>200</width>
         <height>30</height>
        </size>
       </property>
       <item>
        <property name="text">
         <string>正常显示</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>只显示1层</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="7" column="0">
      <widget class="QLabel" name="label_14">
       <property name="minimumSize">
        <size>
         <width>120</width>
         <height>30</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>120</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>是否隐藏</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="7" column="1">
      <widget class="QComboBox" name="isHide">
       <property name="minimumSize">
        <size>
         <width>200</width>
         <height>30</height>
        </size>
       </property>
       <item>
        <property name="text">
         <string>否</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>是</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="7" column="2" colspan="2">
      <widget class="QLabel" name="label_10">
       <property name="minimumSize">
        <size>
         <width>120</width>
         <height>30</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>120</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>Script</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="7" column="4">
      <widget class="QLineEdit" name="script">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="minimumSize">
        <size>
         <width>200</width>
         <height>30</height>
        </size>
       </property>
       <property name="text">
        <string>Buff</string>
       </property>
      </widget>
     </item>
     <item row="8" column="0">
      <widget class="QLabel" name="label_11">
       <property name="minimumSize">
        <size>
         <width>120</width>
         <height>30</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>120</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>循环时间</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="8" column="1">
      <widget class="Tool_NumberEdit" name="totaltime">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="minimumSize">
        <size>
         <width>200</width>
         <height>30</height>
        </size>
       </property>
       <property name="value">
        <number>1</number>
       </property>
      </widget>
     </item>
     <item row="8" column="2" colspan="2">
      <widget class="QLabel" name="label_12">
       <property name="minimumSize">
        <size>
         <width>120</width>
         <height>30</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>120</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>持续时间</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="8" column="4">
      <widget class="Tool_NumberEdit" name="looptime">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="minimumSize">
        <size>
         <width>200</width>
         <height>30</height>
        </size>
       </property>
       <property name="value">
        <number>1</number>
       </property>
      </widget>
     </item>
     <item row="2" column="2" colspan="3">
      <widget class="AffixTipCBox" name="affixTip">
       <property name="minimumSize">
        <size>
         <width>300</width>
         <height>30</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="Tool_ListEdit" name="seid">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>30</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>16777215</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="3" column="2" colspan="3">
      <widget class="BuffSeidTipCBox" name="seidTip">
       <property name="minimumSize">
        <size>
         <width>300</width>
         <height>30</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>174</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="2">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Tool_NumberEdit</class>
   <extends>QSpinBox</extends>
   <header location="global">../Tools/tool_numberedit.h</header>
  </customwidget>
  <customwidget>
   <class>Tool_ListEdit</class>
   <extends>QLineEdit</extends>
   <header location="global">../Tools/tool_listedit.h</header>
  </customwidget>
  <customwidget>
   <class>AffixTipCBox</class>
   <extends>QComboBox</extends>
   <header>Affix/affixtipcbox.h</header>
  </customwidget>
  <customwidget>
   <class>BuffSeidTipCBox</class>
   <extends>QComboBox</extends>
   <header>BuffSeid/buffseidtipcbox.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
